### PR TITLE
Order commetns by ASC

### PR DESCRIPTION
**Pull request by tanaka51**
Hi @ohbarye 

This PR makes comments ordering ASC. Now there are ordered by DESC. I think it's useful :)

Please check.
Thanks.

### CODE REVIEW

**Comment by ohbarye** [comment]
Thank you for your awesome PR!

File: index.js [hunk 1]
```diff
@@ -124,7 +124,7 @@ const createSectionSummary = (elements, users) => {
     const reactions = e.reactions.map(r => ` :${r.name}: `.repeat(r.count)).join('');
 
     return `- ${e.content} by ${username} ${reactions}`;
-  }).join('\n')
+  }).reverse.join('\n')
```
Comment (ohbarye) [inline index.js]: @tanaka51 
Can you fix it to `.reverse().join('\n')`?

Comment (tanaka51) [inline index.js]: OH, you're right! Thanks for your nice catch!

**Comment by ohbarye** [comment]
I'm grateful.

**Comment by tanaka51** [comment]
Thank you so much!

**Pull request merged**